code/modules/mob/mob_helpers.dm

```diff
@@ -142,7 +142,7 @@
 ///Takes the mob to shake, the time span to shake for, and the amount of tiles we're allowed to shake by in tiles
 ///Duration isn't taken as a strict limit, since we don't trust our coders to not make things feel shitty. So it's more like a soft cap.
 /proc/shake_camera(mob/M, duration, strength=1)
-	if(!M || !M.client || duration < 1)
+	if(!istype(M) || !M.client || duration < 1)
```

Comment:
I don't like this. we want errors if this happens, why silence them?

Reply:
i think i figured out a good way to do it, let me know if you have a better idea